src/CustomPasswordExpiredController.js

```diff
@@ -35,6 +35,9 @@ function (Okta, FormController, FormType, Footer, Util) {
         else if (expiringSoon && numDays === 0) {
           return Okta.loc('password.expiring.today', 'login');
         }
+        else if (expiringSoon && (numDays === null || numDays === undefined)) {
```

Comment:
Do we need to check if ```numDays``` is null or underfined? Wouldn't it be enough to just check if ```expiringSoon``` is true?  (same comment for the other controller)
```
else if (expiringSoon) {
```

Reply:
Good point, we only need to check `expiringSoon`, I believe. I checked both expired and non-expired cases to make sure.

Reply:
👍 Thanks Michael and Mau. I'll change that